acquia_lift.module

```diff
@@ -60,16 +70,20 @@ function acquia_lift_toolbar() {
         'aria-pressed' => 'false',
       ],
     ],
-    '#access' => AccessResult::allowedIfHasPermission(\Drupal::currentUser(), 'access acquia lift links'),
+    '#access' => \Drupal::currentUser()->hasPermission('access acquia lift links'),
+    '#cache' => [
+      'keys' => ['toolbar'],
+      'contexts' => ['user.permissions'],
```

Comment:
I was challenged by cache context before. Instead of "user.permissions", have you tried just "user" or "user.permissions:access acquia lift links"?